### DISCUSSION TITLE
Need advice on how to make small code change in Android App (personal build)

### DISCUSSION THREAD

**Discussion by jackduckworth** [standardnotes/mobile#528] [Q&A]
I'd like to make a small change to the Android App. It's similar to #20, which is still open for discussion, but I sense it is unlikely to be adopted any time soon.

Basically I'd like the app to use a monospace font for displaying/editing notes. I don't need an option to switch font, I'd just like to do it automatically.

I have managed to build the App, and made changes in a couple of obvious places, where it mentioned Roboto, but it didn't seem to work. I'm thinking the current editor is just using the default font, and I need to add code somewhere to specify a different font. I'm stumped as to where I need to add this code.

Any advice?

(Btw I'm puzzled why this is not the default behaviour. I like to use spaces to do formatting in my notes. It's very basic but it works in just about every text editor, on every platform. It's also much more natural when looking at code snippets, which are generally aligned using spaces.)

**Comment by atmoio** [comment]
Our plain editor in React Native uses a native library for accessing the underlying native textview: https://github.com/standardnotes/sn-textview

So you'd have to fork this library and change it in there, then edit your fork of the mobile app to use your forked textview.

**Comment by jackduckworth** [comment]
Thanks for the advice.

That doesn't look too bad actually. I'll give it a go.
